airscan.go

```diff
@@ -328,6 +372,10 @@ func (c *Client) Scan(settings *ScanSettings) (*ScanState, error) {
 	}, nil
 }
 
+func (c *Client) GetEndpoint(s string) string {
+	return fmt.Sprintf("http://%s%s", c.host, s)
```

Comment:
This can just use string concatenation: "http://" + c.host + s

Reply:
I actually wanted to remove repetition shall we ever want to change protocol from http to https

Reply:
I don’t mind the removal of repetition (introducing the GetEndpoint() method), I was just saying that instead of fmt.Sprintf(), we can just use string concatenation.

BTW, GetEndpoint() doesn’t need to be exported, does it? Make it getEndpoint() for now.

Reply:
Oh I see. Then sure, I totally agree with you there on both things. I'll not export `GetEndpoint` and switch to string concatenation instead of `fmt.Sprintf`